sdk/src/nonce_state.rs

```diff
@@ -0,0 +1,545 @@
+use serde_derive::{Deserialize, Serialize};
+use crate::{
+    account::{Account, KeyedAccount},
+    account_utils::State,
+    hash::Hash,
+    instruction::InstructionError,
+    nonce_program::id,
+    nonce_instruction::{NonceError},
+    pubkey::Pubkey,
+    sysvar::recent_blockhashes::RecentBlockhashes,
+    sysvar::rent::Rent,
+};
+use std::collections::HashSet;
+
+#[derive(Debug, Default, Serialize, Deserialize, PartialEq, Clone, Copy)]
+pub struct Meta {
```

Comment:
Why does `meta` exist if it only contains one element?  Named type for clarity?

Reply:
Named type and old habits die hard 🙂

Reply:
Are you planning to remove `Meta`?

Reply:
Nope. https://github.com/solana-labs/solana/issues/7305